packages/kbn-unified-data-table/src/components/data_table_document_selection.tsx

```diff
@@ -182,19 +244,37 @@ export function DataTableDocumentToolbarBtn({
           iconType="documents"
           onClick={toggleSelectionToolbar}
           data-selected-documents={selectedDocs.length}
-          data-test-subj="dscGridSelectionBtn"
+          data-test-subj="unifiedDataTableSelectionBtn"
           isSelected={isFilterActive}
           badgeContent={selectedDocs.length}
+          css={{
+            '.euiDataGridToolbarControl__badge': {
+              backgroundColor: tint(euiTheme.colors.success, 0.3),
+            },
+          }}
```

Comment:
Not enough contrast for dark theme

<img width="178" alt="Screenshot 2024-04-03 at 14 44 31" src="https://github.com/elastic/kibana/assets/1415710/de577746-777c-4c26-b9aa-6075b4c178c4">

Reply:
Good catch! Updated here: https://github.com/elastic/kibana/pull/166577/commits/31df11b01c18946d23f02117a1525ed1979e9a5d. Ideally I wouldn't be manually setting the badge styles, but we don't allow setting badge props on `EuiDataGridToolbarControl`, so it would need an EUI enhancement.